core/state.go

```diff
@@ -400,9 +400,7 @@ func (s *State) updateContractStorages(stateTrie *trie.Trie, diffs map[felt.Felt
 	for key := range diffs {
 		keys = append(keys, key)
 	}
-	sort.SliceStable(keys, func(i, j int) bool {
-		return len(diffs[keys[i]]) > len(diffs[keys[j]])
-	})
+	slices.SortStableFunc(keys, func(a, b felt.Felt) int { return a.Cmp(&b) })
```

Comment:
I think we should revert because right now sorting is on key values, but before it was
sorting on `diff[key]` length